### PR TITLE
Slovenian

### PR DESCRIPTION
**Pull request by max-ima**
Slovenian translation

### CODE REVIEW

File: language/sl/ucp.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+*
+* @package phpBB Extension - Crizzo Max-width switch - English
```
Comment (Crizz0) [inline language/sl/ucp.php]: I thought this is the slovenian version from 2022. Please update.

**Review by Crizz0 — CHANGES_REQUESTED** [review]

**Comment by max-ima** [comment]
This is the latest version of translation 1.1.0. But where can I get a newer version then.

**Comment by Crizz0** [comment]
You just need to Update the Header of your file. Nothing more. 😀

File: language/sl/ucp.php (hunk 1 above)
Comment (Crizz0) [reply language/sl/ucp.php]:
```suggestion
* @package phpBB Extension - Crizzo Max-width switch - Slovenia
```

File: language/sl/ucp.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* @copyright (c) 2014 Crizzo - www.crizzo.de
```
Comment (Crizz0) [inline language/sl/ucp.php]:
```suggestion
* @copyright (c) 2022 Crizzo - www.crizzo.de
```

**Comment by max-ima** [comment]
Updated

**Pull request closed without merging**